file_adapter.py

```diff
@@ -1,28 +1,52 @@
 """File Adapter for Fake Data."""
 
 import json
-from release_data_abc import ReleaseData
+from release_data_abc import Release, Task, ReleaseData
 
 
 class FileAdapter(ReleaseData):
   """Constructor takes two files: release data and task data"""
 
   def __init__(self, releases_file, tasks_file):
-    json_data = open(releases_file).read()
-    self.releases = json.loads(json_data)
-    json_data = open(tasks_file).read()
-    self.tasks = json.loads(json_data)
+    with open(releases_file) as f:
+      json_releases = f.read()
+
+    releases_dict = json.loads(json_releases)
+    self.releases = {}
```

Comment:
Maybe move the json to release code in the Release class. ie create a from_json method